__tests__/app.test.js

```diff
@@ -179,14 +208,16 @@ describe('/api/articles/:article_id/comments', () => {
             descending: true,
             coerce: true
           });
-        for (const key in comments) {
-            expect(comments[key]).toHaveProperty("comment_id");
-            expect(comments[key]).toHaveProperty("votes");
-            expect(comments[key]).toHaveProperty("created_at");
-            expect(comments[key]).toHaveProperty("author");
-            expect(comments[key]).toHaveProperty("body");
-            expect(comments[key]).toHaveProperty("article_id");
-          };
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id:  expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              article_id:  expect.any(Number)
```

Comment:
Since you're making requests for a specifc article_id, this is the one property you can assert exactly instead of just checking the datatype